### DISCUSSION TITLE
CD-ROM not found with MS-DOS 6.22

### DISCUSSION THREAD

**Discussion by Vort** [bochs-emu/Bochs#102] [Q&A] [answered]
Hello.

I was using `teac_cdi.sys` driver with DOS for a long time.
It always worked for me, both on real hardware and with virtual machines.

But I can't get it to work with Bochs for some reason.
Not sure if because of some my mistake or because of bugs in Bochs.

Can anyone look at my virtual machine ([bochs_dos_cd.zip](https://github.com/bochs-emu/Bochs/files/12995200/bochs_dos_cd.zip)) and say what's wrong with CD support there?

![bochs_no_cd](https://github.com/bochs-emu/Bochs/assets/1242858/d1d999ab-da50-43a3-bd10-e16d2999e6b3)

**Comment by stlintel** [comment]
Have it worked in Bochs before for you, some previous version may be ?

**Reply by Vort** [reply]
I tested with 2.3.7 - same result. Probably it never worked with Bochs.

Looks like these errors may be related to the problem:
`00313080588e[HD    ] IO read(0x01f0) with drq == 0: last command was 00h`
`00337796783e[HD    ] IO read(0x0170) with drq == 0: last command was 00h`

**Comment by stlintel** [comment, marked as answer]
Converting to an issue
